src/views/on_event_view.rs

```diff
@@ -203,6 +203,16 @@ impl<T: View> OnEventView<T> {
             },
         );
     }
+
+    /// Gets access to the inner view.
+    pub fn get_inner(&self) -> &T {
+        &self.inner
```

Comment:
This is not `inner_getters!` becuase view here is defined as:
`self.inner`
and in the rest of `ViewWrapper` it's
`self.view`

Reply:
Let's rename it then! Either that or name all the others `inner`. No real preference here, but renaming it to `view` means fewer changes.

Reply:
I've created seperate PR: https://github.com/gyscos/Cursive/pull/195